qa-heal.planx-pla.net/metadata/aggregate_config.json

```diff
@@ -264,17 +266,17 @@
                 "10.3886/ICPSR30122.v5": {
                     "__manifest": [
                         {
-                            "md5sum": "7cf87ce47b91e3a766753222bc22d098",
-                            "file_name": "TEDS-D-2018-DS0001-bndl-data-spss.zip",
-                            "file_size": 69297783,
-                            "object_id": "dg.H35L/208f4c52-771e-409a-b810-4bcba3c03c51",
+                            "md5sum": "ce2cc5391b9a3d7e794a3b125df250ab",
+                            "file_name": "ICPSR 30122/30122.zip",
+                            "file_size": 1218573774,
+                            "object_id": "dg.H35L/e8cd31d6-dbcb-4006-b7e7-53e34b9d1d73",
                             "commons_url": "externaldata.healdata.org"
                         }
                     ],
                     "data_availability": "available",
                     "authz": "/programs/open",
                     "source_url": "https://www.icpsr.umich.edu/web/ICPSR/studies/30122",
-                    "dataset_1_title": "TEDS-D-2018-DS0001-bndl-data-spss.zip"
+                    "dataset_1_title": "ICPSR 30122/30122.zip"
                 },
                 "XX.XXXX/ICPSRXXXXX.vX": {
```

Comment:
I think we should remove this, can you help?

Reply:
Can do, should I remove 'dataset_1_title' for all of the manifests?

Reply:
no we only need to remove the entire entry of `XX.XXXX/ICPSRXXXXX.vX` itself